### PR TITLE
feat(Alert): Updated design for Alert

### PR DESCRIPTION
**Pull request by jlowery457**
screen shot 2018-05-09 at 7 34 15 pm

### CODE REVIEW

File: scss/components/alert.scss [hunk 1]
```diff
@@ -4,142 +4,114 @@
...
+    );
+
     width: $alert__width;
     max-height: 19.25rem;
-    min-height: rem-calc(72);
-    padding: $alert__padding 0 $alert__padding $alert__padding-left;
-    margin: 0 0 6px;
-    color: $alert__font-color;
-    border: 0;
+    min-height: rem-calc(78);
```
Comment (bfbiggs) [inline scss/components/alert.scss]: Should height be done with rem-calc

File: scss/components/alert.scss [hunk 2]
```diff
@@ -4,142 +4,114 @@
...
+
+      &:before {
+        color: $alert__font-color--info;
+
+        @include icon-info_24;
+      }
+    }
+
+    .#{$alert__class}__content {
+      display: flex;
```
Comment (bfbiggs) [inline scss/components/alert.scss]: could be done with flex mixin

File: scss/components/alert.scss [hunk 3]
```diff
@@ -4,142 +4,114 @@
...
+      flex-direction: column;
+      width: 15rem;
+      padding-right: rem-calc(10);
+      padding-left: rem-calc(16);
     }
 
     .#{$alert__class}__title {
       font-size: $alert__font-size--title;
-      line-height: 32px;
+      line-height: 24px;
```
Comment (bfbiggs) [inline scss/components/alert.scss]: this seems like should be done with rem-calc

File: scss/settings/alerts.scss [hunk 4]
```diff
@@ -22,16 +21,17 @@ $alert__background-color--warning: $yellow-base !default;
...
 
-$alert__padding: .75rem;
-$alert__padding--small: 0.5rem;
-$alert__padding-left: 5.5rem;
+$alert__padding-right: rem-calc(13);
+$alert__padding-left: rem-calc(17);
 $alert__padding-bottom: 10px;
 
-$alert__width: 25rem;
+$alert__width: 21.375rem;
```
Comment (bfbiggs) [inline scss/settings/alerts.scss]: Same width comment as above.

File: scss/components/alert.scss (hunk 2 above)
Comment (jlowery457) [reply scss/components/alert.scss]: I tried, but the flex-flow part of that mixin blows up my ellipsis stuff for MeetingAlert.

**Comment by bfbiggs** [comment]
Is the X not centered because of the additional pixels?

**Comment by jlowery457** [comment]
The X is not centered due to the button having display: inline-block set in button-base() mixin.  Changing that to inline-flex fixes it, but breaks the loading button.

File: scss/components/alert.scss (hunk 2 above)
Comment (bfbiggs) [reply scss/components/alert.scss]: Hmm usually an issue with overflow setting.

File: scss/components/alert.scss [hunk 5]
```diff
@@ -4,141 +4,136 @@
...
-      margin: 0;
+    width: $alert__width;
+    max-height: rem-calc(308);
+    min-height: rem-calc(78);
+    padding: $alert__padding $alert__padding-right $alert__padding $alert__padding-left;
+    margin-bottom: rem-calc(8);
+    border: 1px solid $black-12;
+    box-shadow: 0 8px 16px 0 $black-08;
+
+    .#{$alert__class}__icon {
```
Comment (bfbiggs) [inline scss/components/alert.scss]: Any reason not to add &__ to all of these ?

File: scss/components/alert.scss [hunk 6]
```diff
@@ -4,141 +4,136 @@
...
+
+    .#{$alert__class}__button {
+      margin-left: rem-calc(8);
     }
 
     &.#{$alert__class}--success {
-      &:before {
-        @include icon-check-circle_24;
-        background-color: $alert__background-color--success;
+      .#{$alert__class}__icon {
```
Comment (bfbiggs) [inline scss/components/alert.scss]: Would be nice to have a color/icon mixin for all these

**Pull request merged**